File: app/src/main/java/kr/djspi/pipe01/dto/Pipe.java
package kr.djspi.pipe01.dto;

import java.io.Serializable;

import lombok.Data;

@Data
public class Pipe implements DataItem, Serializable {

    public int id = -1;
    public int type_id;
    public int spi_id;
    public double depth;
    public String material;
    public int supervise_id;
    public String supervise_contact;
    public String construction;
    public String construction_contact;
}

File: app/src/main/java/kr/djspi/pipe01/dto/Pipe.kt
package kr.djspi.pipe01.dto

import java.io.Serializable

data class Pipe(var id: Int = -1) : DataItem, Serializable {

    var type_id: Int? = null
    var spi_id: Int? = null
    var depth: Double? = null
    var material: String? = null
    var supervise_id: Int? = null
    var supervise_contact: String? = null
    var construction: String? = null
    var construction_contact: String? = null
}
